src/sentry/static/sentry/app/utils/measurements/measurements.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+type Measurement = {
+  name: string;
+  key: string;
+};
+
+type MeasurementCollection = {[key: string]: Measurement};
+
+const MEASUREMENTS: MeasurementCollection = {
+  'measurements.fp': {name: 'First Paint', key: 'measurements.fp'},
```

Comment:
This is okay for now, but i think we'll want to define these in the `discover/fields.tsx` or load them via an end point.